server/datastore/mysql/migrations/tables/20230926082152_AddWindowsMDMTables.go

```diff
@@ -0,0 +1,70 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20230926082152, Down_20230926082152)
+}
+
+func Up_20230926082152(tx *sql.Tx) error {
+	_, err := tx.Exec(`
+CREATE TABLE windows_mdm_pending_commands (
+    -- command uuid is fleet-generated, so that a command has a unique id as
+    -- soon as it is enqueued. Note that a UUID is 36 chars, but for
+    -- consistency with nano_mdm_commands we use 127 here.
+    command_uuid    VARCHAR(127) NOT NULL,
+    device_id       VARCHAR(255) NOT NULL,
+
+    -- verb is e.g. Add, Exec, Get, etc.
+    cmd_verb        VARCHAR(20)  NOT NULL,
+    setting_uri     VARCHAR(255) NOT NULL,
+    setting_value   VARCHAR(255) NOT NULL,
+    data_type       VARCHAR(255) NOT NULL,
+    system_origin   TINYINT(1) NOT NULL DEFAULT FALSE,
+
+    created_at      TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    updated_at      TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+
+    PRIMARY KEY (command_uuid, device_id)
```

Comment:
Should we include a secondary index on `device_id` alone? I see that we filter only on that when listing pending commands, for example.

Reply:
I think that's a good idea. I will look for examples within our codebase on where a secondary index was added to follow the same pattern

Reply:
Alternatively, if you will only filter by `command_uuid` in combination with `device_id`, you can reverse the order of the primary key to place `device_id` first.

Reply:
@mna @gillespi314 I had some issues adding the secondary index. and addressing the suggestion above. Would it be possible for you to tackle these changes as part of the API endpoints story? (Those endpoints will end up queuing cmds into the `windows_mdm_pending_commands` table.

Reply:
> Would it be possible for you to tackle these changes as part of the API endpoints story?
@marcosd4h For sure, no problem.

Reply:
@mna thanks! I don't have much experience dealing with DB/storage logic and feel that it's better is someone else take this part